lib/core/src/display/render/css3d/html/html_renderer.rs

```diff
@@ -63,23 +65,33 @@ fn set_object_transform(dom: &JsValue, matrix: &Matrix4<f32>) {
     }
 }
 
-fn setup_camera_transform
-(dom:&JsValue, near:f32, half_width:f32, half_height:f32, matrix:&Matrix4<f32>) {
+fn setup_camera_perspective
+(dom:&JsValue, y_scale:f32, half_width:f32, half_height:f32, matrix:&Matrix4<f32>) {
```

Comment:
Why we are making this change? Why `near` is named `y_scale` now? I may be missing some information here. I jsut don't understand the meaning of `y_scale`.

Reply:
Naming it `near` was misleading. We need to translate camera's Z by `y_scale`.

Reply:
We need to describe it better, as discussed on Discord.